crates/anvil/core/src/eth/transaction/mod.rs

```diff
@@ -401,6 +401,14 @@ pub fn to_alloy_transaction_with_hash_and_sender(
     }
 }
 
+fn v_parity_to_u64(v: Parity) -> u64 {
+    match v {
+        Parity::Eip155(v) => v,
+        Parity::NonEip155(b) => (b as u64) + 27,
+        Parity::Parity(b) => b as u64,
+    }
+}
```

Comment:
@prestwich is this something we should add as function instead?

Reply:
yes, in [this file](https://github.com/alloy-rs/core/blob/main/crates/primitives/src/signature/parity.rs)

Reply:
Actually, to method `to_u64()` already existed ; I completely missed it :see_no_evil: Updated the PR!